File: fix_onboarding.js
const fs = require('fs');
let code = fs.readFileSync('app/src/main/java/com/example/ui/screens/OnboardingScreen.kt', 'utf8');

const regex = /premiumFeatures = listOf\([\s\S]*?\)/m;
const replacement = `premiumFeatures = listOf(
                "Guardado de Drafts e Historial de Partidas",
                "Repertorio de Campeones Favoritos",
                "Personalización de Temas Visuales",
                "Colección de Avatares Exclusiva",
                "Diseños de Bordes Exclusivos",
                "Partículas Mágicas (Barra de Navegación)"
            )`;
code = code.replace(regex, replacement);

fs.writeFileSync('app/src/main/java/com/example/ui/screens/OnboardingScreen.kt', code);

File: fix_onboarding.py
import re

with open('app/src/main/java/com/example/ui/screens/OnboardingScreen.kt', 'r') as f:
    content = f.read()

bad_str = """            highlights = listOf(
                tr("Historial de drafts y estadísticas de Win Rate personal"),
                tr("Sistema para añadir campeones a tu lista de favoritos"),
                tr("Cambio de temas visuales exclusivos (Noxus, Jonia, Vacío)"),
                tr("Selección y desbloqueo de Avatares de Runaterra"),
                tr("Análisis táctico y Coaching Challenger avanzado")
            )"""

good_str = """            highlights = listOf(
                tr("Historial de drafts y estadísticas de Win Rate personal"),
                tr("Sistema para añadir campeones a tu lista de favoritos"),
                tr("Cambio de temas visuales exclusivos (Noxus, Jonia, Vacío)"),
                tr("Selección y desbloqueo de Avatares de Runaterra")
            )"""

content = content.replace(bad_str, good_str)

with open('app/src/main/java/com/example/ui/screens/OnboardingScreen.kt', 'w') as f:
    f.write(content)
